packages/azure-container-services/src/activate.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * (c) Copyright IBM Corp. 2023
+ */
+
+'use strict';
+
+const instanaCore = require('@instana/core');
+const { backendConnector, consoleLogger } = require('@instana/serverless');
+
+const identityProvider = require('./identity_provider');
+
+const { tracing, util: coreUtil } = instanaCore;
+const { normalizeConfig } = coreUtil;
+
+let logger = consoleLogger;
+
+const config = normalizeConfig({});
+
+function init() {
+  if (process.env.INSTANA_DEBUG || process.env.INSTANA_LOG_LEVEL) {
+    logger.setLevel(process.env.INSTANA_DEBUG ? 'debug' : process.env.INSTANA_LOG_LEVEL);
+  }
+  if (!process.env.WEBSITE_OWNER_NAME && !process.env.WEBSITE_SITE_NAME && !process.env.WEBSITE_RESOURCE_GROUP) {
+    logger.error(
+      'Initializing @instana/azure-container-services failed. The environment variables are not set.' +
+        'This container instance will not be monitored.'
+    );
+    return;
+  }
+  try {
+    identityProvider.init();
+    backendConnector.init(identityProvider, logger, false, true, 950);
+    instanaCore.init(config, backendConnector, identityProvider);
+    tracing.activate();
```

Comment:
TBD: Why do we have no metrics for this package? We do have metrics for Fargate and GCP?

Reply:
Could we please add a comment here which explains why we have no metrics right now?
Fargate & Azure works differently and the agent pulls all the data we need for now.

Reply:
added